cli/src/config/templates.toml

```diff
@@ -166,6 +166,12 @@ if(hidden,
 )
 '''
 
+# Uncomment the following or put it in your config to enable facier graph node
+# styling.
+#
+# log_node = 'builtin_log_node'
+# op_log_node = 'builtin_op_log_node'
+
```

Comment:
I expect the current state is temporary, and the builtin node templates will be enabled in the next release.

> Let me know if you prefer that I document this in config.md.

Perhaps, some generalized doc can be added to `templates.md`? Something like
* to override the default, set `templates.<name>` where `<name>` is usually a command name
* e.g. `templates.log = 'builtin_log_comfortable'`
* stock templates are usually named as `builtin_..`
* the list can be obtained by `jj log -T` (without template value)

Reply:
This sounds like a good idea. I'll look into it.

This PR was meant for the short term, but I'm fine with not merging it; it should serve it's audience even unmerged.

> the list can be obtained by `jj log -T` (without template value)

That's really nice, good to know!